src/main/java/eu/bbmri_eric/negotiator/negotiation/state_machine/negotiation/NegotiationLifecycleService.java

```diff
@@ -25,6 +25,7 @@ public interface NegotiationLifecycleService {
    *
    * @return the new status of the Negotiation
    */
-  NegotiationState sendEvent(String negotiationId, NegotiationEvent negotiationEvent)
```

Comment:
leave the orgiginal signature, just add a new method with update signature. Then the parameter will indeed be optional